### ISSUE TITLE
Add link text to ParsedLinks

### ISSUE THREAD

**Issue by truencoa** [sjdirect/abot#185]
It would be nice to have the text of anchor, we added this as Core.Uri.Text (inherited from System.Uri)
```
namespace Abot.Core
{
    public class Uri : System.Uri
    {
        public Uri(string uriString) : base(uriString) { }
        public Uri(string uriString, UriKind uriKind) : base(uriString, uriKind) { }
        public Uri(System.Uri baseUri, string relativeUri) : base(baseUri, relativeUri) { }
        public Uri(System.Uri baseUri, System.Uri relativeUri) : base(baseUri, relativeUri) { }
        public string Text { get; set; }
        public override string ToString() { return base.ToString().EndsWith("/") ? base.ToString().Remove(base.ToString().Length - 1, 1) : base.ToString(); }
    }
}
```

...but the latest version was too difficult to refactor it back in.  Perhaps a new Poco class of Link?

This was added to HyperLinkParser.GetUris

`                    if (_cleanURLFunc != null)
                        newUri = new Uri(_cleanURLFunc(newUri.AbsoluteUri)) { Text = hrefValue.Value };`

**Comment by chen198328** [comment]
good

**Comment by sjdirect** [comment]
Should be fixed with nuget version 2.0.57 or greater. HyperLink class now has RawHrefText property. See https://github.com/sjdirect/abot/commit/466c1b30b969d3fa2b289c7a195b71bcc975af84
